### ISSUE TITLE
finish-args-unnecessary-xdg-data-access

### ISSUE THREAD

**Issue by Bollos00** [flathub/flatpak-builder-lint#110]
My app (LibreMines) needs access to the directory `~/.local/share/libremines` so it can save scores, user configs and load custom minefield themes.

For that, I used the `finish-arg`: `"--filesystem=xdg-data/libremines:create",`.

For some reason, the build is failing on the part of check manifest, as shown [here](https://buildbot.flathub.org/#/builders/5/builds/1932).

```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions io.github.Bollos00.LibreMines.json
 in dir /srv/buildbot/worker/build-x86_64-4/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions io.github.Bollos00.LibreMines.json'
 using PTY: False
{
    "errors": [
        "finish-args-unnecessary-xdg-data-access"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
program finished with exit code 1
elapsedTime=0.588067
```

I do not understand the issue, should [this file be changed like this?](https://github.com/flathub/flatpak-builder-lint/blob/a4377c5278f7193a485bfc1b500b47c197c35e8f/flatpak_builder_lint/staticfiles/exceptions.json#L1443)

**Comment by hfiguiere** [comment]
Either the app use XDG properly and then data is saved in the app specific location (and will persist for the next time the app is launched), or the app doesn't use XDG and then you can add `--persist=.local/share/libremines` to the finish-args.

**Comment by j8r** [comment]
@hfiguiere what about shortcuts, if `~/.local/share/applications` doesn't exist, for some reason (new system for example), is it fine to give `--filesystem=xdg-data/applications/MyApp::create`?
